src/pages/pricing.tsx

```diff
@@ -228,7 +228,7 @@ const PricingPage: React.SFC<{}> = () => (
                                 Gitpod is free for up to 50 hours/month on any public repository. 
                             </p>
                             <p>
-                                If you're a professional open-source developer and need more hours, let us know, by applying for a <strong> free Unlimited Open Source Plan</strong>. Your GitHub / GitLab track record should clearly show your regular open-source contributions.
+                                If you're a professional open-source developer and need more hours, you can apply to our <strong> free Unlimited Open Source Plan</strong>.
```

Comment:
Please don't add extra spaces within tags like `<strong>` and `</strong>`, because this can lead to double-spaces in the text (there are already spaces outside of these tags).

Also, I think it's helpful to clearly call out the full **Unlimited Open Source** plan name, without any extra words in bold or caps, so that you very visually see & remember exactly what you want (i.e. not just "the open source plan" as many users currently request, or "hey I want to use Gitpod for open source thanks", etc).

In summary:

```suggestion
                                If you're a professional open-source developer and need more hours, you can apply to our free <strong>Unlimited Open Source</strong> plan.
```